packages/@aws-cdk/aws-glue/test/table.test.ts

```diff
@@ -1600,6 +1600,37 @@ test('can specify a physical name', () => {
   });
 });
 
+test('storage descriptor parameters', () => {
+  const app = new cdk.App();
+  const stack = new cdk.Stack(app, 'Stack');
+  const database = new glue.Database(stack, 'Database');
+  new glue.Table(stack, 'Table', {
+    database,
+    columns: [{
+      name: 'col',
+      type: glue.Schema.STRING,
+    }],
+    dataFormat: glue.DataFormat.JSON,
+    storageParameters: [
+      { key: glue.StorageParameters.SKIP_HEADER_LINE_COUNT, value: '2' },
+      { key: glue.StorageParameters.custom('separatorChar'), value: ',' },
+      { key: glue.StorageParameters.COMPRESSION_TYPE, value: glue.CompressionType.GZIP },
```

Comment:
This is really nice, good work!

Reply:
It was the only way I saw around that issue I was having previously.